akka-actor/src/main/scala/akka/dispatch/Mailboxes.scala

```diff
@@ -202,8 +206,14 @@ private[akka] class Mailboxes(
         // It doesn't matter if we create a mailbox type configurator that isn't used due to concurrent lookup.
         val newConfigurator = id match {
           // TODO RK remove these two for Akka 2.3
-          case "unbounded" => UnboundedMailbox()
-          case "bounded"   => new BoundedMailbox(settings, config(id))
+          case "unbounded"                               => UnboundedMailbox()
+          case "bounded"                                 => new BoundedMailbox(settings, config(id))
+          case _ if id.startsWith(BoundedCapacityPrefix) =>
+            // hack to allow programmatic set of capacity through props in akka-typed but still share
+            // mailbox configurators for the same size
+            val capacity = id.split(':')(1).toInt
```

Comment:
I guess it's ok for us if that fails with a cryptic exception in case we or someone else doesn't follow the format.

Reply:
Yeah, it's not public API in any way